src/get-data-args.ts

```diff
@@ -8,6 +8,10 @@ export function getComponentDataArgs<Services extends object>(
     const { componentRegistrar } = getRegistrars(layout)
     const componentDataDefinition = componentRegistrar.get(componentType)
 
+    // This can be undefined
```

Comment:
Can or can't?

Reply:
Can be, another PR coming for json-react-layouts to fix the typings

Reply:
https://github.com/sevenwestmedia-labs/json-react-layouts/pull/23